runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -2989,7 +2994,8 @@ outOfProcessCompilationEnd(
                                      logFileStr, svmSymbolToIdStr,
                                      (resolvedMirrorMethodsPersistIPInfo) ?
                                                          std::vector<TR_ResolvedJ9Method*>(resolvedMirrorMethodsPersistIPInfo->begin(), resolvedMirrorMethodsPersistIPInfo->end()) :
-                                                         std::vector<TR_ResolvedJ9Method*>()
+                                                         std::vector<TR_ResolvedJ9Method*>(),
+                                     *entry->_optimizationPlan
```

Comment:
alignment seems a bit off here

Reply:
I feel like it should be like this, the above two lines were indented because of the ternary expression.

Reply:
Yeah, I misread the code